plugins/client.go

```diff
@@ -0,0 +1,59 @@
+package plugins
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"strings"
+
+	"github.com/docker/docker/utils"
+)
+
+const (
+	versionMimetype = "appplication/vnd.docker.plugins.v1+json"
+)
+
+func NewClient(addr string) *Client {
+	// FIXME: TLS? :scream:
```

Comment:
agreeeeeed. this was @calavera's comment i left in here for discussion like this. :)